fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java

```diff
@@ -549,24 +548,26 @@ private Statistics computeFilter(Filter filter) {
     }
 
     private ColumnStatistic getColumnStatistic(TableIf table, String colName) {
-        if (totalColumnStatisticMap.get(table.getName() + colName) != null) {
-            return totalColumnStatisticMap.get(table.getName() + colName);
-        } else if (isPlayNereidsDump) {
-            return ColumnStatistic.UNKNOWN;
-        } else {
-            long catalogId;
-            long dbId;
-            try {
-                catalogId = table.getDatabase().getCatalog().getId();
-                dbId = table.getDatabase().getId();
-            } catch (Exception e) {
-                // Use -1 for catalog id and db id when failed to get them from metadata.
-                // This is OK because catalog id and db id is not in the hashcode function of ColumnStatistics cache
-                // and the table id is globally unique.
-                LOG.debug(String.format("Fail to get catalog id and db id for table %s", table.getName()));
-                catalogId = -1;
-                dbId = -1;
+        long catalogId;
+        long dbId;
+        try {
+            catalogId = table.getDatabase().getCatalog().getId();
+            dbId = table.getDatabase().getId();
+        } catch (Exception e) {
+            // Use -1 for catalog id and db id when failed to get them from metadata.
+            // This is OK because catalog id and db id is not in the hashcode function of ColumnStatistics cache
+            // and the table id is globally unique.
+            LOG.debug(String.format("Fail to get catalog id and db id for table %s", table.getName()));
+            catalogId = -1;
+            dbId = -1;
+        }
+        if (isPlayNereidsDump) {
```

Comment:
i think the better way to do this is put all column stats in dump file into cache first. and then we don't need to change any of the code here

Reply:
we do not have Env.getCurrentEnv() when replaying minidump, so it can not load to cache